### PR TITLE
teams: migrated index page to vue components

### PR DESCRIPTION
**Pull request by vitoravelino**
I'll investigate how we can improve the bootstrap of the components in another PR to not block the current work on this one.

### CODE REVIEW

File: app/controllers/teams_controller.rb [hunk 1]
```diff
@@ -33,13 +38,22 @@ def create
...
     @team.owners << current_user
 
-    if @team.save
-      @team.create_activity(:create,
-                            owner:      current_user,
-                            parameters: { team: @team.name })
-      respond_with(@team)
-    else
-      respond_with @team.errors, status: :unprocessable_entity
+    respond_to do |format|
```
Comment (vitoravelino) [inline app/controllers/teams_controller.rb]: @mssola Should I move this to the API too?

Comment (mssola) [reply app/controllers/teams_controller.rb]: I was thinking on it but I didn't want to annoy you more with API thingies 😄 I don't have that much spare time in SUSEcon, but I'll try to open up a PR with it asap 👍

Comment (vitoravelino) [reply app/controllers/teams_controller.rb]: Same comment as the other. 😃

File: lib/api/v1/teams.rb [hunk 2]
```diff
@@ -22,6 +22,30 @@ class Teams < Grape::API
           present policy_scope(Team), with: API::Entities::Teams
         end
 
+        desc "Create a team",
```
Comment (vitoravelino) [inline lib/api/v1/teams.rb]: @mssola Please review this!

**Pull request merged**